src/metrics.go

```diff
@@ -63,6 +63,8 @@ func getMetrics(client *gojmx.Client) (map[string]interface{}, map[string]map[st
 						if len(visitedColumnFamilies) < args.ColumnFamiliesLimit {
 							visitedColumnFamilies[eventkey] = struct{}{}
 						} else {
+							log.Warn("Visited column families limit reached. Current limit set to %d",
```

Comment:
two small questions:
 - can we add the column family name that we are dropping?
 - if not would it make sense to log it just once instead of N times each run?
 
Now that we have the context, any chance to quickly improve this `for{for{else{if{if{if{else{}}}}}}` for example creating a function?

Reply:
The second one is clearly out of scope, feel free to ignore it completely 😅

Reply:
If I am not wrong `columnfamily` is what we need. Updated the log message here 868f12d05a6fa5d0c178a1c7dffbbe3e42511c7e.